### PR TITLE
Fix cursor position after language box insertion

### PR DESCRIPTION
**Pull request by ptersilie**
This commit fixes another bug that happens when a language box is
inserted and results in the cursor jumping to the wrong location
afterwards.

### CODE REVIEW

File: lib/eco/treemanager.py [hunk 1]
```diff
@@ -241,8 +241,11 @@ def find_next_visible(self, node):
     def find_previous_visible(self, node, cross_lang=True):
         """Return the previous visible node in the parse tree."""
         if self.is_visible(node):
-            node = node.previous_terminal() # XXX check for multiterm
-        while not self.is_visible(node):
+            if type(node.symbol) is MagicTerminal and cross_lang:
+                node = node.symbol.ast.children[-1]
+            else:
+                node = node.previous_terminal() # XXX check for multiterm
```
Comment (ltratt) [inline lib/eco/treemanager.py]: Is the `XXX` still relevant?

Comment (ptersilie) [reply lib/eco/treemanager.py]: Actually, it is not. Fixed in latest commit.

**Comment by ltratt** [comment]
Please squash.

**Comment by ptersilie** [comment]
Squashed!

**Review by ltratt — APPROVED** [review]

**Pull request merged**